### ISSUE TITLE
Anime reviews always returns empty list

### ISSUE THREAD

**Issue by fbaierl** [SandroHc/reactive-jikan#13]
After upgrading to v2 of reactive-jikan, the anime reviews query is always returning an empty list. Am I doing something wrong here:

```
    public List<Review> getAnimeReviews(int malId, int page){
        try {
            final var result = _jikan.query().anime().reviews(malId)
                    .page(page)
                    .execute()
                    .retryWhen(Retry.backoff(BACKOFF_ATTEMPTS, java.time.Duration.ofMillis(BACKOFF_MILLIS)))
                    .collect(Collectors.toList())
                    .doOnError(e -> Log.e(TAG, "Unable to search for reviews for anime with id: " +
                            malId + ". page: " + page + ". " + e.toString()))
                    .onErrorResume(a -> Mono.just(new ArrayList<>()))
                    .block();
            return result;
        } catch (JikanQueryException e) {
            Log.e(TAG, "Unable to search for reviews for anime with id: " +
                    malId + ". page: " + page, e);
            return new ArrayList<>();
        }
    }
```

I also tried the same mal id as in the [test](https://github.com/SandroHc/reactive-jikan/blob/master/src/test/java/net/sandrohc/jikan/query/anime/AnimeReviewsQueryTest.java):

<img width="492" alt="image" src="https://user-images.githubusercontent.com/38532657/202862724-38b18b39-6633-4765-8281-c3319049efaf.png">

**Comment by SandroHc** [comment]
Looks like an issue on the Jikan API side; issue is being tracked on jikan-me/jikan-rest#281. Should fix itself once jikan-me/jikan-rest#285 lands.

**Comment by fbaierl** [comment]
Alright, I'll close this then. Thank you very much for the fast reply and the info.
